engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/hv/EmailValidator.java

```diff
@@ -29,25 +29,31 @@
  *
  * @author Emmanuel Bernard
  * @author Hardy Ferentschik
+ * @author Guillaume Smet
  */
 public class EmailValidator implements ConstraintValidator<Email, CharSequence> {
-	private static final String ATOM = "[a-z0-9!#$%&'*+/=?^_`{|}~-]";
-	private static final String DOMAIN = ATOM + "+(\\." + ATOM + "+)*";
+	private static final String LOCAL_PART_ATOM = "[a-z0-9!#$%&'*+/=?^_`{|}~\u0080-\uFFFF-]";
```

Comment:
Not really.

The local part accepts this new range of character as valid.

The domain is first converted to ASCII using IDN.toASCII (which verifies that everything is legit, UTF-8, label length and so on), then is validated with the regexp.